simplq/src/components/common/SidePanel/SidePanelItem.jsx

```diff
@@ -4,14 +4,8 @@ import { Collapse } from '@material-ui/core';
 import PropagateLoader from 'react-spinners/PropagateLoader';
 import styles from '../../../styles/sidePanel.module.scss';
```

Comment:
Replace PropogateLoader with LoadingIndicator. It's the same thing.

Reply:
Done